canonical-web-and-design/ubuntu.com
# Build MicroCerts table with mock data

## Done

- Add mock data to `views.py`
- Add new `_pattern-cube-table.scss` file
- Fix order of layout imports in `styles.scss` to be in alphabetical order, and fix `utility_animations` filename to follow underscore convention
- Build MicroCerts table following design, with 3 examples of modules: _Passed_, _Failed_ and _Never tried_

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube/microcerts
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [design](https://app.zeplin.io/project/5fc7a466de658e0cb99276c8/screen/5fc7a4e598710d09c2e8b79a)

## Issue / Card

Fixes [#8855](https://github.com/canonical-web-and-design/ubuntu.com/issues/8855)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/101618166-8414bb80-3a09-11eb-911c-95ecb5acaa52.png)
